.travis.yml

```diff
@@ -4,50 +4,41 @@ cache:
   directories:
   - "$HOME/.composer/cache"
 
+php:
+  - '7.1'
+  - '7.2'
+  - '7.3'
+
+env:
+  - ENUM=2 LARAVEL=5.4
+  - ENUM=2 LARAVEL=5.5
+  - ENUM=2 LARAVEL=5.6
+  - ENUM=2 LARAVEL=5.7
+  - ENUM=2 LARAVEL=5.8
+  - ENUM=3 LARAVEL=5.4
+  - ENUM=3 LARAVEL=5.5
+  - ENUM=3 LARAVEL=5.6
+  - ENUM=3 LARAVEL=5.7
+  - ENUM=3 LARAVEL=5.8
```

Comment:
This is a great improvement :+1:

Reply:
thanks took me some commits to get it right :-)